src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -148,7 +148,8 @@ def collect_data(geop_result, geojson):
     z['UrbAreaTotal'] = sum(z['Area'][NRur:])
     z['PhosConc'] = phosphorus_conc(z['SedPhos'])
 
-    z['NormalSys'] = normal_sys(z['Area'])
+    normal_sys_estimate = int(num_normal_sys(z['Area']))
```

Comment:
Consider moving this and the array formation within `num_normal_sys`, so that here we have only

```python
z['NumNormalSys'] = num_normal_sys(z['Area'])
```

Every other line in this method populates `z` directly.